frontend/src/components/PsychiatristList.tsx

```diff
@@ -0,0 +1,60 @@
+// components/PsychiatristList.tsx
+import React from 'react';
+import BookMark from '../assets/bookmark.svg'
+import Message from '../assets/message.svg'
+
+enum Gender {
+  Male = 0,
+  Female = 1
+}
+
+interface Psychiatrist {
+  id: number;
+  first_name: string;
+  last_name: string;
+  title: string;
+  profile_pic: null;
+  availability: string[];
+  gender: Gender;
+  location: string;
+  language: string[];
+  specialty: string[];
+  description: string
+}
+
+
+interface PsychiatristListProps {
+  results: Psychiatrist[];
+}
+
+const PsychiatristList: React.FC<PsychiatristListProps> = ({ results }) => {
```

Comment:
love the code comments and how they make this huge div more readable. might be even better to add spacing to segment the code even more for readability for the future devs working on this file